### PR TITLE
Sort the imports in the split configurations

### PR DESCRIPTION
**Pull request by fwyzard**
#### PR description:

When generating a split configuration, sort the import in each file by section and by module name.

For example, before:
```python
from ..modules.l1tPFPuppiHT_cfi import *
from ..modules.l1tPFPuppiHT450off_cfi import *
from ..sequences.HLTParticleFlowSequence_cfi import *
from ..sequences.HLTAK4PFPuppiJetsReconstruction_cfi import *
from ..modules.hltPFPuppiHT_cfi import *
from ..modules.hltPFPuppiHT1050_cfi import *
```
after:
```python
from ..modules.hltPFPuppiHT_cfi import *
from ..modules.hltPFPuppiHT1050_cfi import *
from ..modules.l1tPFPuppiHT_cfi import *
from ..modules.l1tPFPuppiHT450off_cfi import *
from ..sequences.HLTAK4PFPuppiJetsReconstruction_cfi import *
from ..sequences.HLTParticleFlowSequence_cfi import *
```

#### PR validation:

None.

### CODE REVIEW

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
looks like Python 3 removed support for tuple unpacking via PEP 3113 - good job guys

**Comment by fwyzard** [comment]
please test

**Comment by smuzaffar** [comment]
please test
lets tests with newer IB which has the needed data external

File: FWCore/ParameterSet/python/Config.py [hunk 1]
```diff
@@ -227,7 +227,6 @@ def defaultOptions_():
                               forceEventSetupCacheClearOnNewRun = untracked.bool(False),
                               throwIfIllegalParameter = untracked.bool(True),
                               printDependencies = untracked.bool(False),
-                              deleteNonConsumedUnscheduledModules = untracked.bool(True),
```
Comment (makortel) [inline FWCore/ParameterSet/python/Config.py:230]: Why this change?

File: FWCore/ParameterSet/python/Config.py [hunk 2]
```diff
@@ -1991,9 +1992,7 @@ def __init__(self,*arg,**args):
 
         def testProcessDumpPython(self):
             self.assertEqual(Process("test").dumpPython(),
-"""import FWCore.ParameterSet.Config as cms
-
-process = cms.Process("test")
+"""import FWCore.ParameterSet.Config as cms\n\nprocess = cms.Process("test")
```
Comment (makortel) [inline FWCore/ParameterSet/python/Config.py:1995]: Why this change?

File: FWCore/ParameterSet/python/Config.py [hunk 3]
```diff
@@ -2011,11 +2010,10 @@ def testProcessDumpPython(self):
     SkipEvent = cms.untracked.vstring(),
     allowUnscheduled = cms.obsolete.untracked.bool,
     canDeleteEarly = cms.untracked.vstring(),
-    deleteNonConsumedUnscheduledModules = cms.untracked.bool(True),
```
Comment (makortel) [inline FWCore/ParameterSet/python/Config.py:2014]: Why this change?

File: FWCore/ParameterSet/python/Config.py [hunk 4]
```diff
@@ -2011,11 +2010,10 @@ def testProcessDumpPython(self):
...
     emptyRunLumiMode = cms.obsolete.untracked.string,
     eventSetup = cms.untracked.PSet(
         forceNumberOfConcurrentIOVs = cms.untracked.PSet(
-            allowAnyLabel_=cms.required.untracked.uint32
+
```
Comment (makortel) [inline FWCore/ParameterSet/python/Config.py:2016]: Why this change?

**Comment by fwyzard** [comment]
@makortel those changes likely slipped in by mistake, using an older branch to make the PR - let me clean it up

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
actually, since I'm here, let me see if I can do one better ...

**Comment by fwyzard** [comment]
please test

**Comment by makortel** [comment]
All failures appear to be in the IB already, but I'd prefer one round of tests with the next IB that should clean with at least some of those.

**Comment by silviodonato** [comment]
please test

> All failures appear to be in the IB already, but I'd prefer one round of tests with the next IB that should clean with at least some of those.

**Comment by makortel** [comment]
+1

**Comment by qliphy** [comment]
+1
